### PR TITLE
Fix code styling

### PR DESCRIPTION
**Pull request by siwane**
Hello, this PR is used to demonstrate phpcs fixer apply on taoCe extension. As a prototype, it does not contains version bump. It is using the same phpcs command as `http://qa.kitchen.it.taocloud.org/job/deploy-test-package/job/product-ci%252Ftaoce/lastStableBuild/php-code-sniffer/`

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (mike-ionut-mihai-sandu-tao) [inline manifest.php]: is this line necessary ? 
maybe we can take a look at this config that your using

Comment (siwane) [reply manifest.php]: I'm using [php-cs](https://github.com/squizlabs/PHP_CodeSniffer) with default parameters.
After check, it appears that [our jenkins CI](http://qa.kitchen.it.taocloud.org/job/deploy-test-package/job/product-ci%252Ftaoce/lastStableBuild/php-code-sniffer/folder.1867278147/category.-829123767/source.12fb8437-6324-4e4c-b751-bfa13ada634e/#1) is throwing the same error

Comment (julien-sebire) [reply manifest.php]: PSR-2 says `blank_line_after_php`: no.
https://www.php-fig.org/psr/psr-2/ search for `blank_line_after_php:`

Comment (siwane) [reply manifest.php]: As discussed IRL, it is psr12 interpretation

**Comment by siwane** [comment]
@julien please see, https://github.com/oat-sa/extension-tao-community/pull/209#discussion_r366325382

**Comment by julien** [comment]
> @julien please see, [#209 (comment)](https://github.com/oat-sa/extension-tao-community/pull/209#discussion_r366325382)

not sure what I should do about that

**Pull request merged**